### ISSUE TITLE
Error installing package "chatgpt-azure-tasks"

### ISSUE THREAD

**Issue by TimaxLacs** [deep-foundation/npm-packager#94]
When trying to install the `chatgpt-azure-tasks` package in `promiseThree` install the following error is displayed:
```
{
  "errors": [
    "dependency [@deep-foundation/chatgpt-tokens-gpt-4-encoder TokensDependency], not found"
  ]
}
```
![image](https://github.com/deep-foundation/npm-packager/assets/83237728/08fc56fc-52e0-445c-869a-66bf57cc9428)

**Comment by Konard** [comment]
Here we have to update error message to have it enough context for user to fix the problem.

**Comment by Konard** [comment]
I have made the message more clear at https://github.com/deep-foundation/deeplinks/commit/9c0613f2985599227b5f560cd3ffed67f8461eaa

**Comment by Konard** [comment]
This is the dependency tree:

```
gitpod /workspace/empty (main) $ npm i @deep-foundation/chatgpt-azure-tasks

added 12 packages in 8s
npm notice 
npm notice New minor version of npm available! 10.2.4 -> 10.5.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v10.5.0
npm notice Run npm install -g npm@10.5.0 to update!
npm notice 
gitpod /workspace/empty (main) $ npm list @deep-foundation/chatgpt-tokens-gpt-4-encoder
empty@ /workspace/empty
└─┬ @deep-foundation/chatgpt-azure-tasks@0.0.17
  └─┬ @deep-foundation/chatgpt-azure-templates@0.0.6
    └─┬ @deep-foundation/chatgpt-azure@2.0.0
      └── @deep-foundation/chatgpt-tokens-gpt-4-encoder@1.0.3

gitpod /workspace/empty (main) $ 
```

Looks like to solve the issue it will be required to update `@deep-foundation/chatgpt-azure` package, then update `@deep-foundation/chatgpt-azure-templates`, and then update `@deep-foundation/chatgpt-azure-tasks@0.0.17`. Because it is caused by breaking change, we have to update major versions in all these packages.

**Comment by TimaxLacs** [comment]
bag fixed
